Form-Controls/index.html

```diff
@@ -6,23 +6,74 @@
   <title>My form exercise</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Quicksand:wght@300&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <div>
+      <h1>Product Pick</h1>
+    </div>
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
+  <section>
+    <div class="section__container">
+      <form action="">
+        <fieldset>
+          <div class="name__wrapper">
+            <div>
+                <label for="firstName">First Name *</label>
+                <input type="text" name="" id="firstName" required pattern="[A-Za-z]{3}" title="Must contain at least 3 letters">
+            </div>
+            <div>
+                <label for="lasttName">Last Name *</label>
+                <input type="text" name="" id="lasttName" required pattern="[A-Za-z]{3}" title="Must contain at least 3 letters">
```

Comment:
Is this what the requirements say?

Reply:
There were 2 letters, right? fixed thanks.

Reply:
Not really, I removed it. thanks